tests/filecheck/dialects/kernel/ops.mlir

```diff
@@ -0,0 +1,29 @@
+// RUN: XDSL_ROUNDTRIP
+// RUN: XDSL_GENERIC_ROUNDTRIP
+
+// CHECK: builtin.module {
+// CHECK-GENERIC: "builtin.module"() ({
+
+%0 = "test.op"() : () -> i32
+// CHECK-NEXT:   %0 = "test.op"() : () -> i32
+// CHECK-NEXT-GENERIC:   %0 = "test.op"() : () -> i32
```

Comment:
This should be `CHECK-GENERIC-NEXT:` instead of `CHECK-NEXT-GENERIC`

Reply:
oops! thanks for spotting this!